.github/workflows/yarn-upgrade-v1main.yml

```diff
@@ -26,7 +26,7 @@ jobs:
 
       - name: Locate Yarn cache
         id: yarn-cache
-        run: echo "::set-output name=dir::$(yarn cache dir)"
+        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
```

Comment:
No v1 updates on this branch. Only v2 is updated via `main`. In fact, this file should be deleted altogether on `main`.